dashboard/src/API/apiService.ts

```diff
@@ -91,20 +90,14 @@ class ApiService {
     if (!namespace || !chart || !chart.name || version === undefined)
       return Promise.reject(new Error("missing parameters"));
 
-    const url = `${this.baseUrl}/api/helm/repositories/values?chart=${namespace}/${chart.name}&version=${version}`;
+    const url = `/api/helm/repositories/values?chart=${namespace}/${chart.name}&version=${version}`;
     const response = await fetch(url);
     const data = await response.text();
 
     return data;
   };
 }
 
-let baseURL = "";
```

Comment:
I'm not sure why this code is deleted. It used to work well to support both local dev work and production setup. @nir2002